src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/HttpClientTransport.cs

```diff
@@ -97,6 +97,7 @@ sealed class PipelineRequest : HttpPipelineRequest
             public PipelineRequest()
             {
                 _requestMessage = new HttpRequestMessage();
+                CorrelationId = Guid.NewGuid().ToString();
```

Comment:
Should we be specific about how to write the guid into a string?

Reply:
Correlation id is a transparent value, we can decide on the format we like more but it doesn't matter much.

Reply:
I looked into this a little bit more and the commonly used format seems to be the default guid.ToString() format `client-request-id: 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0)`

Reply:
Added a test for the format.

Reply:
Is there a means to pass an existing ` CorrelationId` to the request?  The pattern that I've typically used/seen is to accept a correlation identifier and if one isn't populated, then generate one for this operation that bubbles down into any related calls.    It could be useful if we're working against a service that requires a sequence of related operations or potentially exposing logging to SDK consumers at some point.